### PR TITLE
Fix/1.5mltubesracks

### PR DESCRIPTION
**Pull request by x-anderson**
Testing the tube racks out in master and on ui resulted in colliisons (wrong y-offset) - have adjusted these to fix.

### CODE REVIEW

File: microArch/factory/make_plate_library.go [hunk 1]
```diff
@@ -332,34 +332,46 @@ func makePlateLibrary() map[string]*wtype.LHPlate {
...
+	ystart = 10.0      // distance from top left side of plate to first well
 	zstart = 6.0       // offset of bottom of deck to bottom of well
 
 	welltype2mleppy := wtype.NewLHWell("2mlEpp", "", "", "ul", 2000, 25, eppy, wtype.LHWBV, 8.2, 8.2, 45, 4.7, "mm")
 
 	plate = wtype.NewLHPlate("Kombi2mlEpp", "Unknown", 4, 2, 45, "mm", welltype2mleppy, wellxoffset, wellyoffset, xstart, ystart, zstart)
 	plates[plate.Type] = plate
 
-	// Eppendorfrack
+	// Eppendorfrack 425 for 2ml tubes
```
Comment (chr1sgrant) [inline microArch/factory/make_plate_library.go]: the description says // Eppendorfrack 425 for 2ml tubes 

but the definition is for 1.5ml tubes

File: microArch/factory/make_plate_library.go [hunk 2]
```diff
@@ -332,34 +332,46 @@ func makePlateLibrary() map[string]*wtype.LHPlate {
...
 
 	wellxoffset = 18.0 // centre of well to centre of neighbouring well in x direction
 	wellyoffset = 18.0 //centre of well to centre of neighbouring well in y direction
-	xstart = 5.0       // distance from top left si z9fdJwde of plate to first well
-	ystart = 5.0       // distance from top left side of plate to first well
-	zstart = 7.0 - 1.0 // offset of bottom of deck to bottom of well
+	xstart = 4.5       // distance from top left side of plate to first well
+	ystart = 9         // distance from top left side of plate to first well
+	zstart = 5.0       // offset of bottom of deck to bottom of well
```
Comment (chr1sgrant) [inline microArch/factory/make_plate_library.go]: did you try these with both heads? 

We should pursue the head specific offset rule

File: microArch/factory/make_plate_library.go (hunk 1 above)
Comment (chr1sgrant) [inline microArch/factory/make_plate_library.go]: It would be worth having definitions for both

Comment (x-anderson) [inline microArch/factory/make_plate_library.go]: Have added both (although i stuck with the 2ml as having the 1.5mls in there with the lids, its not really possibly to have many tubes)

File: microArch/factory/make_plate_library.go (hunk 2 above)
Comment (x-anderson) [inline microArch/factory/make_plate_library.go]: Yes, I verified these with both heads.   i can look into the head specific offset rule.

File: microArch/factory/make_plate_library.go (hunk 1 above)
Comment (chr1sgrant) [inline microArch/factory/make_plate_library.go]: The annotation on line 343 is for 2ml tubes but the plate specified from lines 345 - 354 depends on using 1.5ml eppendorfs. I think there should be both 1.5ml and 2ml dimensions specified for this rack as well as rack 424

Comment (x-anderson) [inline microArch/factory/make_plate_library.go]: Im not sure i understand - line 343 is the annotation for 2ml and line 351 specifies the 2ml plate - similarly lines 354 and 364 for the 1.5ml?

I didnt observe any differences needed in dimensions for using 1.5ml and 2ml tubes so the only differences between these are the volumes.

Comment (chr1sgrant) [inline microArch/factory/make_plate_library.go]: OK sorry, I see that now looking at the whole file, from the changes it was confusing. So the residual volume for the 2ml tube is 25ul and 50ul for the 1.5ml tube?

Comment (x-anderson) [inline microArch/factory/make_plate_library.go]: havent verified the residual volumes (they were already inputted there so I assumed they had been checked previously?). If not I can do this. However Id really like to get this in asap so I can run my workflow ideally today, so if its oK to merge now and I can update the residuals tomorrow/when i get back.

Comment (x-anderson) [inline microArch/factory/make_plate_library.go]: Also to clarify -  these 2ml tubes are the ones with screw caps - I can update the definition to reflect that and push up

**Pull request merged**